baremetal/metal3machine_manager_test.go

```diff
@@ -55,7 +56,8 @@ const (
 	kcpName                   = "kcp-pool1"
 )
 
-var ProviderID = "metal3://12345ID6789"
+var Bmhuid = types.UID("222222-2222-2222-2222-222222222222")
```

Comment:
I know this is just an example but why not use something that looks proper?

Reply:
Thanks, will change that.

Reply:
Done